### PR TITLE
Bug 1788691: Clarify DNS resource record requirements in UPI

### PR DESCRIPTION
**Pull request by jcpowermac**
Provide explict requirements for type of DNS
resource record for:
- Kubernetes api
- Routes
- etcd instances

### CODE REVIEW

File: modules/installation-dns-user-infra.adoc [hunk 1]
```diff
@@ -44,15 +44,15 @@ API calls can fail, and you cannot retrieve logs from Pods.
...
+machines that run the Ingress router pods, which are the worker nodes by
 default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
 
 .2+|etcd
 |`etcd-<index>.<cluster_name>.<base_domain>`
-|{product-title} requires DNS records for each etcd instance to point to the
-control plane machines that host the instances. The etcd instances are
+|{product-title} requires DNS A/AAAA records for each etcd instance to point to the
+control plane machines that host the instances.  The etcd instances are
```
Comment (kalexand-rh) [inline modules/installation-dns-user-infra.adoc]: There's an extra space here. Otherwise, this LGTM!

Comment (jcpowermac) [reply modules/installation-dns-user-infra.adoc]: Thanks for the review.  Fixed the space

**Pull request merged**